change/@fluentui-react-accordion-1a557172-5b53-49bb-b896-5774b84cce18.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Updating based on changes to composition types.",
```

Comment:
It looks like these change files are duplicates of the ones in #20890 and might need to be re-created

Reply:
I've recreated the change files 👍